pages/galerie_connecte.php

```diff
@@ -9,6 +9,9 @@
     <link rel="stylesheet" href="ressources/styles/index.css">
     <link rel="stylesheet" href="ressources/styles/modale.css">
     <link rel="stylesheet" href="ressources/styles/carte_cocktail.css">
+    <link rel="stylesheet" href="../ressources/styles/publication.css">
```

Comment:
Les autres fonctionnent sans le `../`, c'est du détail mais c'est plus simpa que tout soit unis x')
```suggestion
    <link rel="stylesheet" href="ressources/styles/publication.css">
```